Tools/AP_Bootloader/bl_protocol.cpp

```diff
@@ -842,6 +861,30 @@ bootloader(unsigned timeout)
         break;
 #endif
 
+        case PROTO_READ_MULTI: {
+            arg = cin(50);
+            if (arg < 0) {
```

Comment:
is there a chance to add the ability to self read the bootloader section as well?